files/pt-br/mozilla/add-ons/webextensions/api/browsingdata/remove/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: browsingData.remove()
+slug: Mozilla/Add-ons/WebExtensions/API/browsingData/remove
+tags:
+  - API
+  - Add-ons
+  - Extensions
+  - Method
+  - Reference
+  - WebExtensions
+  - browsingData
+  - remove
+browser-compat: webextensions.api.browsingData.remove
+translation-of: Mozilla/Add-ons/WebExtensions/API/browsingData/remove
+---
+{{AddonSidebar()}}
+
+Remove dado de navegação específico.
+
+O dado de navegação a ser removido está especificado na opção  `dataTypes`, em que é um objeto {{WebExtAPIRef("browsingData.DataTypeSet")}}.
+
+Você pode usar a opção `removalOptions`, um {{WebExtAPIRef("browsingData.RemovalOptions")}} objeto, para controlar o quão longe no passado a remoção de dados será feita e se deseja remover dados apenas de páginas normais ou remover  de aplicativos e extensões hospedados também.
+
+Isso é uma função assíncrona que retorna uma [`Promessa`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
+
+## Sintaxe
+
+```js
+let removing = browser.browsingData.remove(
+  removalOptions,            // RemovalOptions object
+  dataTypes                  // DataTypeSet object
+)
+```
+
+### Parâmetros
+
+- `removalOptions`
+  - : `object`. Um objeto {{WebExtAPIRef("browsingData.RemovalOptions")}}, que pode ser usado para controlar o quão longe no passado a remoção de dados será feita e se deseja remover dados apenas de páginas normais ou remover  de aplicativos e extensões hospedados também.
+- `dataTypes`
+  - : `object`. Um objeto {{WebExtAPIRef("browsingData.DataTypeSet")}}, descrevendo os tipos de dados para remover (e.g. histórico, downloads, ...).
+
+### Valor retornado
+
+Uma [`Promessa`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) que será realizada sem argumentos quando a remoção terminar. Se ocorrer algum erro, a promessa será rejeitada com uma mensagem de erro.
+
+## Exemplos
+
+Remove o histórico de download e navegação da última semana:
+
+```js
+function onRemoved() {
+  console.log("removed");
+}
+
+function onError(error) {
+  console.error(error);
+}
+
+function weekInMilliseconds() {
+  return 1000 * 60 * 60 * 24 * 7;
+}
+
+let oneWeekAgo = (new Date()).getTime() - weekInMilliseconds();
+
+browser.browsingData.remove(
+  {since: oneWeekAgo},
+  {downloads: true, history: true}).
+then(onRemoved, onError);
+```
+
+Remove todo o histórico de download e navegação:
+
+```js
+function onRemoved() {
+  console.log("removed");
+}
+
+function onError(error) {
+  console.error(error);
+}
+
+browser.browsingData.remove({},
+  {downloads: true, history: true}).
+then(onRemoved, onError);
```

Comment:
```suggestion

let oneWeekAgo = new Date().getTime() - weekInMilliseconds();

browser.browsingData
  .remove({ since: oneWeekAgo }, { downloads: true, history: true })
  .then(onRemoved, onError);
```